examples/pie_and_polar_charts/nested_pie.py

```diff
@@ -0,0 +1,63 @@
+"""
+Nested pie charts
+=================
+
+The following examples show how two ways to build a nested pie chart
+in Matplotlib.
+
+The most straightforward way to build a pie chart is to use the
```

Comment:
I would make this part a comment (instead of a docstring) for symmetry with below.